src/EA.Weee.Web/Areas/Admin/Views/Obligations/UploadObligations.cshtml

```diff
@@ -6,47 +6,50 @@
     ViewBag.Title = "Manage PCS obligations";
 }
 
-@(this.WeeeGds().BackLink(@Url.UrlFor<ObligationsController>(a => a.SelectAuthority())))
+@section full_main_content
+{
+    @(this.WeeeGds().BackLink(@Url.UrlFor<ObligationsController>(a => a.SelectAuthority())))
 
-<header class="hgroup">
-    <div>
-        <h1 class="govuk-heading-l">Manage PCS obligations</h1>
-    </div>
-</header>
+    <header class="hgroup">
+        <div>
+            <h1 class="govuk-heading-l">Manage PCS obligations</h1>
+        </div>
+    </header>
 
-@using (Html.BeginForm("UploadObligations", "Obligations", FormMethod.Post))
-{
-    @Html.AntiForgeryToken()
-    @Html.HiddenFor(m => m.Authority)
-    <p>
-        To upload your PCS's obligations, you need to first download the CSV template for the current compliance year. After this you can fill in the template with the data and upload it.
-    </p>
-    <p>
-        <button aria-hidden="true" class="link-submit govuk-link govuk-link--no-visited-state" type="submit">Download template</button>
-    </p>
+    @using (Html.BeginForm("UploadObligations", "Obligations", FormMethod.Post))
+    {
+        @Html.AntiForgeryToken()
+        @Html.HiddenFor(m => m.Authority)
+        <p>
+            To upload your PCS's obligations, you need to first download the CSV template for the current compliance year. After this you can fill in the template with the data and upload it.
+        </p>
+        <p>
+            <button aria-hidden="true" class="link-submit govuk-link govuk-link--no-visited-state" type="submit">Download template</button>
```

Comment:
do you need to add tabIndex = -1?

Reply:
do you need to add this class too class="govuk-visually-hidden" ?

Reply:
Probably remove the aria-hidden attribute as that hides it for screen readers.

Reply:
ignore my comments then

Reply:
removed attribute